src/types.hpp

```diff
@@ -356,7 +361,7 @@ namespace madrona_gpudrive
         }
     };
 
-    const size_t TrajectoryExportSize = 2 * 2 * consts::kTrajectoryLength + 2 * consts::kTrajectoryLength + ActionExportSize * consts::kTrajectoryLength;
+    const size_t TrajectoryExportSize = 2 * 2 * consts::maxEpisodeLength + 2 * consts::maxEpisodeLength + ActionExportSize * consts::maxEpisodeLength;
```

Comment:
This looks incorrect. Even if you increase the episode length, the data shape will always remain the same